### PR TITLE
14 metrik export healthstatus der clients

### PR DESCRIPTION
**Pull request by nbinderHHN**

### CODE REVIEW

File: metrics/healtstatus.go
Comment (nbinderHHN) [inline metrics/healtstatus.go:1]: i am not sure if health_status.go would be better

File: metrics/healtstatus.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// RecordHealthStatusEntry
+// Records the health status of a client.
+// uses the same senderLabel as signal_strength.go
+func RecordHealthStatusEntry(senderClientName string, healthStatus float64) {
```
Comment (nbinderHHN) [inline metrics/healtstatus.go:11]: I would use name the param like this:
```suggestion
func RecordHealthStatusEntry(clientName string, healthStatus float64) {
```

File: metrics/healtstatus.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	signal.With(prometheus.Labels{senderLabel: senderClientName}).Set(healthStatus)
+}
+
+var status = promauto.NewGaugeVec(
+	prometheus.GaugeOpts{Name: "health_status", Help: "Health status of a client"},
+	[]string{senderLabel})
```
Comment (nbinderHHN) [inline metrics/healtstatus.go:17]: I would also use the clientLabel like in #20

**Review by Elias970 — APPROVED** [review]
good work simon hehe

**Pull request merged**